Glossary A~F

cironunes opened the pull request.

cironunes committed:
translate(glossary): introduction + A~C

- annotation
- attribute directives
- bootstrap
- component

willmendesneto commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -26,6 +35,8 @@ include _util-fns
   :marked
     In practice a synonym for [Decoration](#decorator).
+
+    Na prática, um sinônimo de Decoration.
```
o `[Decoration](#decorator)` continua neste texto, pois está como um link para outro tópico.

cironunes replied:
boa

willmendesneto commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -36,8 +47,15 @@ include _util-fns
     The `ngClass` directive for adding and removing CSS class names is a good example of
     an Attribute Directive.
+
+    A diretiva ngClass para adicionar e remover classes CSS é um bom exemplo 
```
A palavra "ngClass" tem uma diferença de ênfase, então continua como "A diretiva `ngClass` ..."

cironunes replied:
?

Foxandxss committed:
chore: add README.md to each plunker
closes #1974

cironunes committed:
translate(glossary): dash-case, data binding

willmendesneto commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -94,10 +112,18 @@ include _util-fns
 
+    Pode referir-se a [Injeção de dependencias](#dependency-injection)
+    binding entre um "token" ou uma "chave" e o provedor da dependencia.
+    Este caso mais raro deve ser ter o contexto claro.
```
`deve ser ter` não passa uma idéia realmente. Neste caso uma boa opção seria `deve ter o contexto claro` ou a frase poderia ser reformulada para algo do tipo `Neste caso em específico o contexto deve ser o mais claro possível`

cironunes replied:
Boa! Copy'n paste me quebrou nessa.

thelgevold committed:
docs(cb-dynamic-forms): update subtitle
closes #2004

cironunes committed:
docs(glossary): improve DI word
closes #2007

cironunes committed:
translate(glossary): decoration, DI, directive, ES

chalin committed:
docs(toh-6): search query fixes
closes #2008
Changes:
- Drop `asObservable()` since it is deprecated in RxJS 5 (see the
[migration
guide](https://github.com/ReactiveX/RxJS/blob/master/MIGRATION.md#operat
ors-renamed-or-removed)).
- Drop `+` from hero search query URL: `app/heroes/?name=${term}+`. At
best it is interpreted as a regex op that serves no purpose, at worst,
it gets interpreted as a space (cf. [HTML 4.01 section
17.13.4](https://www.w3.org/TR/REC-html40/interact/forms.html#h-17.13.4)
).
- Rename `searchSubject` to `searchTerms`
- Other minor tweaks to prose.

This work is in preparation for #1924.

Note: toh-6 tests pass.

adaojunior committed:
docs(server-communication): Resolve todo (#1993)

* use latest version of http_in_memory_web_api

* remove todo and drop the ?? body term

chalin committed:
docs(DI/dart): fix incorrect snippet (#2017)

Dart counterpart to #2001.
Fixes #1956 for Dart.

chalin committed:
docs(api/dart): avoid duplicate entries and fix breadcrumbs (#1945)

chalin committed:
docs(architecture): give example of libraries in TS and Dart (#2015)

Follow-up to
https://github.com/angular/angular.io/pull/1972#issuecomment-235738082

chalin committed:
docs(toh-6): add hero search to Dart; minor edits to TS (#2018)

* docs(toh-6/dart): add hero search

Fixes #1924.

* docs(toh-6/ts): minor updates

* post-review updates

* post-review updates

brunoravanhani commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -36,8 +47,15 @@ include _util-fns
     other HTML elements, attributes, properties, and components. They are usually represented
     as HTML attributes, hence the name.
 
+    Uma categoria de Directive que pode escutar e modificar o comportamento de outros 
```
Faltou o link na palavra Directive

brunoravanhani commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -131,16 +170,29 @@ include _util-fns
+    O Componente é uma das peças mais importantes no sistema Angular. 
+    Ele é de fato uma [Directiva](#directive) Angular com um [Template](#template).
+
+    The developer applies the `#{atSym}Component` !{decorator} to
```
Não traduziu aqui

brunoravanhani commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -94,10 +112,18 @@ include _util-fns
     This more rare usage should be clear in context.
 
+    Pode referir-se a [Injeção de dependências](#dependency-injection)
+    binding entre um "token" ou uma "chave" e o provedor da dependência.
```
Faltou o link no `provedor`

brunoravanhani commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -107,8 +133,13 @@ include _util-fns
 
+    O processo de iniciar uma aplicação Angular é chamado "bootstrapping" e é feito usando o método
+    `bootstrap`. Esse método identifica um [Componente](#component) raiz e opcionamente registra 
+    [provedores](#provedores) de serviços usando o [sistema de injeção de dependências](#dependency-injection).
```
Traduziu o link também aqui

kwalrath committed:
docs(dart): update angular.io/dart's link to change log
